examples/kitchen-sink/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# @turnkey/kitchen-sink
+
+## 0.1.1
+
+### Patch Changes
+
+- Updated dependencies [3dd74ac]
+- Updated dependencies [1e36edf]
+- Updated dependencies [4df8914]
+  - @turnkey/sdk-browser@1.9.0
+  - @turnkey/sdk-server@1.6.0
+  - @turnkey/http@2.16.0
+  - @turnkey/sdk-react@1.0.14
```

Comment:
I don't think this is correct, our examples all use workspace versions. We didn't update dependencies, they're already updated :)